charts/truefoundry/templates/tfy-nginx-proxy/_helpers.tpl

```diff
@@ -0,0 +1,276 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "tfy-nginx-proxy.name" -}}
+{{- default "tfy-nginx-proxy" .Values.tfyNginxProxy.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+  Create a default fully qualified app name.
+  We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+  If release name contains chart name it will be used as a full name.
+  */}}
+{{- define "tfy-nginx-proxy.fullname" -}}
+{{- if .Values.tfyNginxProxy.fullnameOverride }}
+{{- .Values.tfyNginxProxy.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default "tfy-nginx-proxy" .Values.tfyNginxProxy.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+  Create chart name and version as used by the chart label.
+  */}}
+{{- define "tfy-nginx-proxy.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+  Pod Labels
+  */}}
+{{- define "tfy-nginx-proxy.podLabels" -}}
```

Comment:
Modify the pod Labels and pod annotations with the changes done in main chart

Reply:
this can be done pending this PR getting merged - https://github.com/truefoundry/helm-charts/pull/3313